lib/lightning/work_orders.ex

```diff
@@ -355,24 +387,4 @@ defmodule Lightning.WorkOrders do
     )
     |> Repo.one()
   end
-
-  def subscribe(project_id) do
-    Events.subscribe(project_id)
-  end
```

Comment:
Please put this back, I'd like to have callers having to reach into sub-modules for a context. 

You can also use `defdelegate` to forward the call to Events.